state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1107,37 +1108,21 @@ pub mod pallet {
 			refund_params: Option<ChannelRefundParameters>,
 			dca_params: Option<DcaParameters>,
 			// This is only to be checked in the pre-witnessed version (not implemented yet)
-			_boost_fee: BasisPoints,
+			boost_fee: BasisPoints,
```

Comment:
Maybe we add broker fees too? That way we have everything. Engines can default to the empty vec for now.

Reply:
Added broker fees + validation